### ISSUE TITLE
Module Primitive::Symmetric::MAC::HMAC is broken

### ISSUE THREAD

**Issue by brianhuffman** [GaloisInc/cryptol-specs#10]
When pull request #9 was merged, it broke the module `Primitive::Symmetric::MAC::HMAC`, which expects module `Primitive::Keyless::Hash::SHA256` to export a function called `SHA256`.
```
Loading module Primitive::Symmetric::MAC::HMAC
[error] at ./Primitive/Symmetric/MAC/HMAC.cry:34:37--34:43
    Value not in scope: SHA256
[error] at ./Primitive/Symmetric/MAC/HMAC.cry:34:44--34:50
    Value not in scope: SHA256
[error] at ./Primitive/Symmetric/MAC/HMAC.cry:34:51--34:57
    Value not in scope: SHA256
```

**Comment by brianhuffman** [comment]
It looks like #9 also broke module `Primitive::Symmetric::Cipher::Block::SHACAL`:
```
[error] at Primitive/Symmetric/Cipher/Block/SHACAL.cry:67:34--67:45
    Value not in scope: SHA256Block`
```

**Comment by marsella** [comment]
This was addressed in 35a09d7.
